iogt/templates/base.html

```diff
@@ -21,11 +21,13 @@
         <meta property="og:image" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-
         {% compress css %}
         <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss' %}">
         {% endcompress %}
         {% block extra_css %}{% endblock %}
+
+        
+        <script src="https://use.fontawesome.com/57021264f5.js"></script>
```

Comment:
I don't think we need to use images, see: http://unicode-table.com/en/sets/arrows-symbols/